pkg/daemon/daemon.go

```diff
@@ -791,7 +795,11 @@ func (dn *Daemon) syncNode(key string) error {
 			return err
 		}
 
-		if err := dn.triggerUpdate(ufc.currentConfig, ufc.desiredConfig, ufc.currentImage, ufc.desiredImage); err != nil {
+		if missingMC, err := dn.triggerUpdate(ufc.currentConfig, ufc.desiredConfig, ufc.currentImage, ufc.desiredImage); err != nil {
+			// if MC was not found, let user know where they can find more info on this.
+			if apierrors.IsNotFound(err) {
+				mcdMissingMC.WithLabelValues(missingMC).Inc()
```

Comment:
This is using the same current/desired config from the above `dn.prepUpdateFromCluster()` call right? If MC was missing, wouldn't the previous `Inc()` already cover this?

Reply:
i added this one in for safety. I think there is a path where we can get to this and not the other. All that matters is that they both can't happen!